wazo_confd/plugins/group/validator.py

```diff
@@ -18,20 +14,12 @@ class GroupValidator(Validator):
     def validate(self, group):
         self.validate_unique_name_through_queue(group)
 
+    # TODO(pc-m): remove this validation once queues get there name generated
     def validate_unique_name_through_queue(self, group):
         queue = queue_dao.find_by(name=group.name)
         if queue is not None:
             raise errors.resource_exists('Queue', name=group.name)
```

Comment:
Can we remove this?

Reply:
I'd rather wait until we do the same thing with queues and avoid someone messing around with queue name and end up doing something that breaks everything and then open a bug.

Reply:
Ok :+1: